### PR TITLE
v2-ui Need host level filter for containers and pods

### PR DESCRIPTION
**Pull request by harishshg**
Fixes # .

Changes proposed in this pull request:
- added hosts filter in container grid view filter 
- added hosts, conainer  filter in pods grid view filter

@deepfence/engineering

### CODE REVIEW

File: deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/ContainersTable.tsx [hunk 1]
```diff
@@ -71,14 +72,18 @@ const loader = async ({ request }: LoaderFunctionArgs): Promise<LoaderData> => {
...
+    [key: string]: string[];
+  } = hosts.length ? { host_name: hosts } : {};
   const searchSearchNodeReq: SearchSearchNodeReq = {
     node_filter: {
       filters: {
         compare_filter: null,
         contains_filter: {
           filter_in: {
             active: [true],
+            ...filterIn,
```
Comment (manV) [inline deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/ContainersTable.tsx:86]: why create filterIn variable, if it is not being reused? we can just put value directly here.

File: deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/PodsTable.tsx [hunk 2]
```diff
@@ -36,14 +38,19 @@ const loader = async ({ request }: LoaderFunctionArgs): Promise<LoaderData> => {
...
+    [key: string]: string[];
+  } = hosts.length ? { host_name: hosts, kubernetes_cluster_name: clusters } : {};
   const searchSearchNodeReq: SearchSearchNodeReq = {
     node_filter: {
       filters: {
         compare_filter: null,
         contains_filter: {
           filter_in: {
             active: [true],
+            ...filterIn,
```
Comment (manV) [inline deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/PodsTable.tsx:53]: same

**Review by manV — CHANGES_REQUESTED** [review]
As we discussed, current SearchableHostList/SearchableClusterList dropdowns operate on nodeId not host/cluster name. but for filter we need host/cluster name in the onChange values. Please make dropdown configurable so that we can choose what property we want to operate on, nodeId or host/cluster_name.

File: deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/ContainersTable.tsx (hunk 1 above)
Comment (harishshg) [reply deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/ContainersTable.tsx]: removed local var and added inline conditional logic

File: deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/PodsTable.tsx (hunk 2 above)
Comment (harishshg) [reply deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/PodsTable.tsx]: removed local var and added inline conditional logic

**Comment by harishshg** [comment]
dynamically send value key as host/cluster_name or node_id.

**Pull request merged**